src/middlewared/middlewared/plugins/service_monitor.py

```diff
@@ -4,17 +4,13 @@
 import threading
 import time
 import tempfile
+import datetime
+import ntplib
 
 from middlewared.service import Service, private
 
 if '/usr/local/www' not in sys.path:
     sys.path.append('/usr/local/www')
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'freenasUI.settings')
-
-import django
-from django.apps import apps
-if not apps.ready:
-    django.setup()
```

Comment:
Are you sure django is really not needed because if any of these freenasUI imports below? It could be django is being loaded in a plugin before this one?